core/internal/entities/token/response.go

```diff
@@ -9,7 +9,7 @@ import (
 type Response struct {
 	TokenID      uuid.UUID `json:"tokenID"`
 	WorkspaceID  uuid.UUID `json:"workspaceID"`
-	RepositoryID uuid.UUID `json:"repositoryID"`
+	RepositoryID string    `json:"repositoryID"`
```

Comment:
maybe we should use the 'omitempty' instead of change the repositoryID response type.